src/main/java/org/kainos/ea/client/DatabaseConnectionException.java

```diff
@@ -0,0 +1,7 @@
+package org.kainos.ea.client;
+
+public class DatabaseConnectionException extends Throwable {
+    public DatabaseConnectionException(Exception e) {
```

Comment:
You don't have a test that this exception is thrown anywhere (if the exception isn't used anywhere, it can be removed)